public/render.js

```diff
@@ -28,7 +28,7 @@ let render = {
 
   welcome: function (response) {
     const message = response.message;
-    const createItemButton =  '<button type="button" class="create-btn">List New Item</button>';
+    const createItemButton =  '<button type="button" class="create-btn">Click Here! <br>List New Item</button>';
```

Comment:
Maybe List New Item was too generic but I feel 'Click Here!' is equally generic if not worse. 
Is there perhaps other phrase we can use?

What are your thoughts?